src/main/java/com/actiontech/dble/backend/datasource/PhysicalDbGroup.java

```diff
@@ -458,7 +458,7 @@ private List<PhysicalDbInstance> getRWDbInstances(boolean includeWrite) {
             if (ds.isAlive() && (!checkSlaveSynStatus() || ds.canSelectAsReadNode())) {
                 okSources.add(ds);
             } else {
-                if (!ds.isAlive()) {
```

Comment:
I think it also need to print log when `!ds.isAlive()`

Reply:
too many logs printed when !ds.isAlive()
inner-1845